lib/rubocop/cop/performance/open_struct.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Performance
+      # This cop checks for `OpenStruct.new` calls.
```

Comment:
I'd expand the rationale here. Keep in mind that this gets published to the RuboCop's manual, so it should give as much details as possible to the end users.